lua/base46/themes/gruvbox.lua

```diff
@@ -52,7 +55,27 @@ M.base_16 = {
   base0F = "#d65d0e",
 }
 
+<<<<<<< HEAD
```

Comment:
I rebased onto maintainer's dev branch, I don't know why wasn't that correctly accounted for.